src/software/backend/input/network/networking/network_client.cpp

```diff
@@ -24,6 +26,8 @@ NetworkClient::NetworkClient(std::string vision_multicast_address,
                               gamecontroller_multicast_port);
 
     startIoServiceThreadInBackground();
+
+    sharedConfig = config;
```

Comment:
This should be in the initializer list for this class, ie:
```
    initial_packet_count(0),
    received_world_callback(received_world_callback),
   sharedConfig(config)
```

https://stackoverflow.com/questions/9903248/initializing-fields-in-constructor-initializer-list-vs-constructor-body

Reply:
Also `sharedConfig` -> `shared_config` as per our naming convention, but could probably just be `config` to be consistent with other usages of configs in the codebase.

Reply:
Now using initializer list and renamed to use config.

Reply:
@garethellis0 can you kindly approve the change? I think this comment is the last thing blocking before I merge so let me know if there are any other problems. Thanks!